src/components/Header/Header.tsx

```diff
@@ -123,8 +112,8 @@ const Header: FC<IHeader> = () => {
 	}, [showHeader]);
 
 	useEffect(() => {
-		setIsGIVeconomyRoute(router.route.startsWith('/giv'));
-		setIsCreateRoute(router.route.startsWith('/create'));
+		setIsGIVeconomyRoute(router.route.startsWith(Routes.GIVECONOMY));
```

Comment:
it should be '/giv'.
if you want to check it by Routes you should check all sub routes